libs/datasets/manual_filter.py

```diff
@@ -96,3 +100,70 @@ def run(dataset: timeseries.MultiRegionDataset, config=CONFIG) -> timeseries.Mul
         dataset = filtered_dataset.append_regions(passed_dataset)
 
     return dataset
+
+
+# Possible values from https://github.com/covid-projections/covid-projections/blob/develop/src/cms-content/region-overrides/region-overrides.json
+_METRIC_TO_FIELDS = {
+    "metrics.caseDensity": common_fields.FIELD_GROUP_TO_LIST_FIELDS[FieldGroup.CASES_DEATHS],
+    # "metrics.infectionRate": common_fields.FieldGroup.
+    "metrics.testPositivityRatio": common_fields.FIELD_GROUP_TO_LIST_FIELDS[FieldGroup.TESTS],
+    "metrics.icuCapacityRatio": common_fields.FIELD_GROUP_TO_LIST_FIELDS[
+        FieldGroup.HEALTHCARE_CAPACITY
+    ],
+    "metrics.vaccinationsInitiatedRatio": common_fields.FIELD_GROUP_TO_LIST_FIELDS[
+        FieldGroup.VACCINES
+    ],
+}
+
+
+def _transform_one_override(
+    override: Mapping, cbsa_to_counties_map: Mapping[Region, List[Region]]
+) -> Mapping:
+    region_str = override["region"]
+    if re.fullmatch(r"[A-Z][A-Z]", region_str):
+        region = Region.from_state(region_str)
+    elif re.fullmatch(r"\d{5}", region_str):
+        region = Region.from_fips(region_str)
+    else:
+        raise ValueError(f"Invalid region: {region_str}")
+
+    include_str = override["include"]
+    if include_str == "region":
+        regions_included = [region]
+    elif include_str == "region-and-subregions":
+        if region.is_state():
+            regions_included = [RegionMask(states=[region.state])]
```

Comment:
`RegionMask` doesn't do anything. It is literally 3 lines: a name, an optional level and optional list of states. I can't say much about its behavior in general because it is highly dependent on the context. I removed the default `level = None` so it must be always explicitly given.